lib/index.js

```diff
@@ -68,22 +75,50 @@ exports.default = function (sails) {
   };
 
   // setup outside like this to allow use of the compiler in http.customMiddleware
-  hook.compiler = (0, _webpack2.default)(config.hook.config, function (err, stats) {
-    if (err) throw err;
-    sails.log.info('sails-hook-webpack: Webpack loaded.');
-    sails.log.silly('sails-hook-webpack: ', stats.toString());
-    if (process.env.NODE_ENV === 'development') {
-      sails.after('lifted', () => {
-        setTimeout(function () {
-          sails.log.info('sails-hook-webpack: Watching for changes...');
-          hook.compiler.watch(config.hook.watchOptions, hook.afterBuild.bind(hook));
-        }, config.watchDelay);
+  if (process.env.NODE_ENV === 'development') {
+      const thread = spawn(function() {});
+      thread.run(function setUpWatch(input, done) {
+
+        var _webpack = require('webpack');
+        var _webpack2 = _interopRequireDefault(_webpack);
+        
+        //change path of webpack config file according to project
+        var config = require('../../../../config/webpack.config.dev.js');
```

Comment:
Can we avoid hardcoding this ?
```
../../../../config/webpack.config.dev.js

```

Reply:
Path is now mentioned in the webpack.config.dev.js file which is extracted and passed as input to the function in the thread.